sonar-iac-plugin/src/main/resources/static/azureresourcemanager_doc.md

```diff
@@ -0,0 +1,22 @@
+---
+title: AzureResourceManager
+key: azureresourcemanager
+---
+
+## Supported Versions
+
+| Format | Version    | Status              |
+|--------|------------|---------------------|
+| JSON   | 2023-06-01 | Partially Supported |
+| Bicep  | any        | Not Supported       |
+
+## Language-Specific Properties
+
+Discover and update the Azure Resource Manager [properties](/analysis/analysis-parameters/) in: **<!-- sonarcloud -->Project <!-- /sonarcloud -->[Administration > General Settings > Languages > AzureResourceManager](/#sonarqube-admin#/admin/settings?category=AzureResourceManager)**
```

Comment:
@nils-werner-sonarsource Could you validate this links, please? I don't have admin access.